tests/unit/test_handlers.py

```diff
@@ -1042,7 +1042,7 @@ def test_set_operation_specific_signer_v4a_existing_signing_context(self):
             context=context, signing_name=signing_name
         )
         # region has been updated
-        self.assertEqual(context['signing']['region'], '*')
+        self.assertEqual(context['signing']['region'], 'abc')
```

Comment:
I think this change is wrong. It's no longer asserting what we're intending and doesn't match the comment. If we're moving this functionality out of `set_operation_specific_signer`, we need another unit or functional test to confirm this behavior remains in place for MRAP.

This also points to a breakage we're making. I'm not sure if there will be any tangible impact from that, but this is a shift in how sigv4a requests would work in our event system.

Reply:
I hope that customers are not currently relying on something they have set manually being overwritten, especially when it's in the signing context which is considered a private interface.  At the moment, all sigv4a endpoint rule-sets will evaluate to `*`.  

There are already [existing tests](https://github.com/boto/botocore/blob/08877232170abe910198e3c1f758ae6d0f2df6a0/tests/functional/test_s3.py#L1234) that confirm this behavior for MRAPs.